job-service-db/src/main/resources/v4.0.0/changelog.xml

```diff
@@ -101,4 +101,38 @@
                                  referencedTableName="job"
                                  referencedColumnNames="partition_id, job_id" />
     </changeSet>
+    <changeSet id="drop_task_stowing_idx_partition_id_and_job_id" author="Gamma Team">
+        <preConditions onFail="CONTINUE">
+            <indexExists schemaName="public"
+                         tableName="stowed_task"
+                         indexName="idx_partition_id_and_job_id"
+                         columnNames="partition_id, job_id"/>
+        </preConditions>
+        <dropIndex indexName="idx_partition_id_and_job_id" tableName="stowed_task"/>
+    </changeSet>
+    <changeSet id="create_task_stowing_idx_partition_id_and_job_id_and_tracking_info_job_task_id" author="Gamma Team">
```

Comment:
Should I also add another changeSet to make this the primary key?

```
<changeSet id="add_task_stowing_primary_key" author="Gamma Team">
    <addPrimaryKey schemaName="public" tableName="stowedTask" columnNames="partition_id, job_id, tracking_info_job_task_id" />
</changeSet>
```

Reply:
That would have further implications for the calling code so you'd be better leaving that for another Jira.